src/components/tvproviders/schedulesdirect.js

```diff
@@ -152,14 +154,20 @@ define(['jQuery', 'loading', 'globalize', 'emby-checkbox', 'listViewStyle', 'emb
                     return i.Id === id;
                 })[0];
                 info.ZipCode = page.querySelector('.txtZipCode').value;
-                info.Country = $('#selectCountry', page).val();
+                info.Country = page.querySelector('#selectCountry').value;
                 info.ListingsId = selectedListingsId;
                 info.EnableAllTuners = page.querySelector('.chkAllTuners').checked;
-                info.EnabledTuners = info.EnableAllTuners ? [] : $('.chkTuner', page).get().filter(function (i) {
-                    return i.checked;
-                }).map(function (i) {
-                    return i.getAttribute('data-id');
-                });
+
+                let enabledTuners = [];
+                if (!info.EnableAllTuners) {
+                    enabledTuners = Array.prototype.filter.call(page.querySelectorAll('.chkTuner'), (tuner) => {
```

Comment:
You can also leverage the spread operator to quickly convert them to an array to call `filter` on.

```suggestion
                    enabledTuners = [...page.querySelectorAll('.chkTuner')].filter((tuner) => {
```

Reply:
Oh, good to know, thanks.